src/ucp/core/ucp_worker.c

```diff
@@ -417,8 +417,13 @@ static unsigned ucp_worker_iface_err_handle_progress(void *arg)
     ucp_ep->am_lane   = 0;
 
     if (ucp_ep_ext_gen(ucp_ep)->err_cb != NULL) {
+        ucs_debug("ep %p: calling user error callback %p with arg %p", ucp_ep,
```

Comment:
minor: pls add `ucs_assert(ucp_ep->flags & UCP_EP_FLAG_USED)` into branch above to clarify that `err_cb` can be set only by API call